packages/core/rollup.config.js

```diff
@@ -35,6 +36,9 @@ export default {
       sourceMap: !production,
       inlineSources: !production
     }),
+    commonjs({
+      include: /node_modules/
```

Comment:
@V-Shadbolt quick question here, can you provide a little background on the reason for this?
Also, can it be scoped to the ustoppabledomains package?
Thanks!

Reply:
@Adamj1232 Yup! This was added because rollup wasn't including the unstoppable domains node module when compiling which threw a bunch of errors when the functions were called.  

The commonjs plugin instructs rollup to convert node modules to ES6 so that they can be interpreted by browsers and the `include: /node_modules/` flag is for the directory of the modules to include during compilation 

I don't believe this can be scoped back unless we can specify just the unstoppable module with something like this:  `include: /node_modules/@unstoppabledomains/`.

Reply:
Ah gotcha, thank you for the fast response! `include: /node_modules/@unstoppabledomains/` is what I had in mind.
I can open a PR or you can but want to get your teams approval on the changes 🤓

Reply:
Hey! Managed to get it rescoped to just Unstoppable using the below snippet while retaining the original functionality of this PR. 

`
commonjs({
      namedExports: {
        '../../node_modules/@unstoppabledomains/resolution/build/index.js': [ 'Resolution' ]
      },
}),`